src/WebJobs.Script/Host/FunctionMetadataProvider.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
```

Comment:
Can you confirm there's no new logic here? This is only what was moved from the metadata manager, correct?

Reply:
Yes everything here was moved from metadata manager. Everything in here is the same. Except for the following 2 things, which we already covered in the previous iteration.
1. Function filtering based on FunctionWhiteList in host.json is still in function metdatamanager
2. Static method from metadataManager has been converted to instance method. WebFunctionManager instead of using static method now uses cached function from metadata from provider.